### PR TITLE
Allow to specify Riemann tags

### PR DESCRIPTION
**Pull request by combor**
## What

This PR allows user to specify custom Rieman taggs so they can be used for metrics filtering. 
 
Tags should be a dictionary under Riemann key in ATC properties. 

```
  - name: atc
    properties:
      riemann:
        tags:
          foo: bar
```
## Why

We like to use disposable environments with one Concourse per env. Each concourse uses Riemann to send metrics to Datadog shared account. It's much easier to filter metrics if tags are set properly.

### CODE REVIEW

File: jobs/atc/spec [hunk 1]
```diff
@@ -284,3 +284,11 @@ properties:
     description: |
       An optional prefix for emitted Riemann services
     default: ""
+
+  riemann.tags:
+    description: |
+      An optional list of tags in key: value format
```
Comment (vito) [inline jobs/atc/spec]: This should probably say 'map' or something rather than 'list'. List to me implies:

```
- env: dev
- foo: bar
```

**Comment by vito** [comment]
Looks good, only one minor rewording requested. Sorry I didn't catch that earlier before having you reopen. :)

**Comment by combor** [comment]
No worries. I'm a little bit confused how new github review functionality works so I replied twice.

**Comment by vito** [comment]
looks good! thanks.

**Pull request merged**